docs/manual/cluster-admin/basic-management-operations.md

```diff
@@ -79,24 +79,31 @@ Generally speaking, PAI services are daemon sets, deployments or stateful sets c
 sudo docker run -itd \
         -e COLUMNS=$COLUMNS -e LINES=$LINES -e TERM=$TERM \
         -v /var/run/docker.sock:/var/run/docker.sock \
-        -v ${HOME}/pai-deploy/cluster-cfg:/cluster-configuration  \
-        -v ${HOME}/pai-deploy/kube:/root/.kube \
         --pid=host \
         --privileged=true \
         --net=host \
         --name=dev-box \
         openpai/dev-box:<openpai version tag>
 ```
 
-You should replace the `<openpai version tag>` with your current OpenPAI version, e.g. `v1.0.0`. In the command, we mount `${HOME}/pai-deploy/kube` to `/root/.kube` in the container. Thus the container has correct config file to access Kubernetes. Also, we mount `${HOME}/pai-deploy/cluster-cfg`, the configuration created by installation, to  `/cluster-configuration` in the container.
+You should replace the `<openpai version tag>` with your current OpenPAI version, e.g. `v1.0.0`.
 
 To use `paictl`, go into the container by:
 
 ```bash
 sudo docker exec -it dev-box bash
 ```
 
-Then, go to folder `/pai`, try to retrieve your cluster id:
+Then, you should manually configure `kubeconfig` in the dev box container. During installation, we have put the correct `kubeconfig` file in `~/pai-deploy/kube/config` on the dev box machine. Please copy its content to `~/.kube/config` in the dev box container:
+
+```bash
+# in the dev box container
+mkdir -p ~/.kube
+# please copy the content in `~/pai-deploy/kube/config` on the dev box machine to `~/.kube/config` here
+vim ~/.kube/config
+```
+
+Go to folder `/pai`, try to retrieve your cluster id:
 
```

Comment:
Need to remind the users of the generated `pai-deploy/cluster-configuration` folder somewhere

Reply:
Done